purchase_order_product_summary/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Purchase Order Product Summary",
+    "summary": """
+        Add a new section with the grouped products and their total quantity. 
```

Comment:
Incomplete summary IMO:
```suggestion
        Adds to purchase report a new section with the grouped products and their total quantity.
```